client/landing/stepper/declarative-flow/internals/steps-repository/sensei-plan/index.tsx

```diff
@@ -5,6 +5,7 @@ import { useCallback, useState } from '@wordpress/element';
 import { __, sprintf } from '@wordpress/i18n';
 import { useI18n } from '@wordpress/react-i18n';
 import classnames from 'classnames';
+import wpcomProxyRequest from 'wpcom-proxy-request';
```

Comment:
I am curious, why did we use this module? 

I am fine with how it is now, just asking because it mentions [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/wpcom-proxy-request/Readme.md) to use [this one](https://github.com/Automattic/wp-calypso/blob/trunk/packages/wpcom.js/README.md) instead. Then there is also [this library ](https://github.com/Automattic/wp-calypso/blob/trunk/packages/wpcom-xhr-request/README.md)

Reply:
Yes. I saw that the `wpcom-proxy-request` suggests to use `wpcom.js` instead.

But `wpcom.js` does not work for our purposes. It is designed to be used mostly with common api endpoints like updating a post or uploading a media image.